src/renderers/testing/ReactTestMount.js

```diff
@@ -20,10 +20,8 @@ var getHostComponentFromComposite = require('getHostComponentFromComposite');
 var instantiateReactComponent = require('instantiateReactComponent');
 var invariant = require('invariant');
 
-import type { ReactElement } from 'ReactElementType';
-
 export type TestRendererOptions = {
-  createNodeMock: (element: ReactElement) => any,
+  createNodeMock: (element: ReactElement<any>) => any,
 };
```

Comment:
This is public API so I needed to use the "public" ReactElement definition provided by Flow. Otherwise tests in www fail.